src/resources/pandoc/datadir/init.lua

```diff
@@ -1,1266 +1,125 @@
-
--- The conversion from UTF-8 to "Windows ANSI codepage" is implemented according to mapping tables published at unicode.org.
--- Mapping tables are stored in human-unreadable compressed form to significantly reduce module size.
-
-local test_data_integrity = false  -- set to true if you are unsure about correctness of human-unreadable parts of this file
-
-local function modify_lua_functions(all_compressed_mappings)
-
-   local function convert_from_utf8(...)   -- for all OS except Windows
-      return ...
-   end
-
-   local char, byte, floor, table_insert, table_concat = string.char, string.byte, math.floor, table.insert, table.concat
-
-   local function decompress_mapping(compressed_mapping)
-
-      local width, offset, base, CS1, CS2, get_next_char = 1.0, 0.0, 0.0, 7^18, 5^22, compressed_mapping:gmatch"%S"
----@diagnostic disable-next-line: unbalanced-assignments
-      local mapping, trees, unicode, ansi, prev_delta_unicode, prev_delta_ansi = {}, {}, 0x7F, 0x7F
-
-      local function decompress_selection(qty, tree)
-         while width <= 94^7 do
-            width, offset, base = width * 94.0, offset * 94.0 + byte(get_next_char()) - 33.0, (base - floor((base + width - 1) / 94^7) * 94^7) * 94.0
-         end
-         if qty then
-            local big_qty = width % qty
-            local small_unit = (width - big_qty) / qty
-            local big_unit = small_unit + 1.0
-            local offset_small = big_qty * big_unit
-            local from, offset_from, left, right
-            if offset < offset_small then
-               width = big_unit
-               offset_from = offset - offset % big_unit
-               from = offset_from / big_unit
-            else
-               width = small_unit
-               offset_from = offset - (offset - offset_small) % small_unit
-               from = big_qty + (offset_from - offset_small) / small_unit
-            end
-            local len, leaf = 1.0, from
-            if tree then
-               leaf, left, right = 4, 0, qty
-               repeat
-                  local middle = tree[leaf]
-                  if from < middle then
-                     right = middle
-                  else
-                     left, leaf = middle, leaf + 1
-                  end
-                  leaf = tree[leaf + 1]
-               until leaf < 0
-               from, len = left, right - left
-               offset_from = left < big_qty and left * big_unit or offset_small + (left - big_qty) * small_unit
-               width = (right < big_qty and right * big_unit or offset_small + (right - big_qty) * small_unit) - offset_from
-            end
-            base, offset = base + offset_from, offset - offset_from
-            CS1, CS2 = (CS1 % 93471801.0) * (CS2 % 93471811.0) + qty, (CS1 % 93471821.0) * (CS2 % 93471831.0) - from * 773.0 - len * 7789.0
-            return leaf
-         end
-         assert((CS1 - CS2) % width == offset)
+if pandoc.system.os == "windows" then
```

Comment:
This is wrong
See https://pandoc.org/lua-filters.html#pandoc.system.os

> The operating system on which the program is running. (string) The most common values are darwin (macOS), freebsd, linux, linux-android, mingw32 (Windows), netbsd, openbsd.

Windows is `mingw32`

````
❯ pandoc lua
Lua 5.4.7  Copyright (C) 1994-2024 Lua.org, PUC-Rio
Embedded in pandoc 3.7.0.1  Copyright (C) 2006-2024 John MacFarlane
> pandoc.system.os                                                                                                                                                        
mingw32
````

Reply:
@tarleb Just for your future reference :)